src/EnergyPlus/RuntimeLanguageProcessor.cc

```diff
@@ -2849,22 +2849,26 @@ namespace RuntimeLanguageProcessor {
 							ShowContinueError( "Blank " + cAlphaFieldNames( 1 ) );
 							ShowContinueError( "Blank entry will be skipped, and the simulation continues" );
 						} else if ( ! errFlag ) {
-							VariableNum = FindEMSVariable( cAlphaArgs( ErlVarLoop ), 0 );
 							// Still need to check for conflicts with program and function names too
 
-							if ( VariableNum > 0 ) {
-								ShowSevereError( RoutineName + cCurrentModuleObject + ", invalid entry." );
-								ShowContinueError( "Invalid " + cAlphaFieldNames( ErlVarLoop ) + '=' + cAlphaArgs( ErlVarLoop ) );
-								ShowContinueError( "Name conflicts with an existing global variable name" );
-								ErrorsFound = true;
+							if ((cCurrentModuleObject.compare("ExternalInterface:FunctionalMockupUnitImport:To:Variable") == 0) && ErlVarLoop > 1){
```

Comment:
Is calling compare any better than just checking equality?  This will also need whitespace fixes to make it uniform with the rest of the code.